Gateway/Request/Header/ExternalPlatformHeaderDataBuilderInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
I'm afraid this interface name might be a bit of a misnomer.
There are more headers defined here, not only `external platform` headers.
Why not call it `HeaderDataBuilderInterface` and keep the name `HeaderDataBuilder` for the builder? that way no need to change `di.xml`

Reply:
Okie, I will change it back. Giving this new interface/builder name was listed as a step in the ticket, so my assumption was that it is discussed and agreed upon in the refinement.